function/services/s3/index.test.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
 
```

Comment:
Additional test is fine, but you're still falling below the coverage threshold on this file.